### ISSUE TITLE
[RCS1264] `roslynator_use_var = when_type_is_obvious` should not report ReadOnlySpan<char> initialization from ""

### ISSUE THREAD

**Issue by sergey-rybalkin** [dotnet/roslynator#1504]
**Product and Version Used**: 

NuGet package v. 4.12.4

**Steps to Reproduce**:

in `.editorconfig`
```
roslynator_use_var = when_type_is_obvious
```
Code:
```csharp
internal class Demo
{
    public static void ReadOnlyCharDemo()
    {
        ReadOnlySpan<char> span = "";
    }
}
```
**Actual Behavior**:

On the `ReadOnlySpan<char>` expression: [RCS1264: Use 'var' or explicit type](https://josefpihrt.github.io/docs/roslynator/analyzers/RCS1264/)

**Expected Behavior**:
Applying suggested code fix to the code above generates `var span = ""` statement which changes `span` variable type to `string`. My guess is that variables declarations with assignment that have implicit type conversion should be ignored by this analyzer.
